prow/config/configs.yaml

```diff
@@ -33,6 +33,9 @@ branch-protection:
               protect: true
             gh-pages:
               protect: true
+              required_pull_request_reviews:
```

Comment:
Can you add a comment here to explain to the future us why this has been done?
I don't want to find myself removing it in two months